packages/antd/src/templates/FieldTemplate/WrapIfAdditionalTemplate.js

```diff
@@ -37,7 +41,9 @@ const WrapIfAdditional = ({
     wrapperCol = VERTICAL_WRAPPER_COL,
     wrapperStyle,
   } = formContext;
-  const { RemoveButton } = registry.templates.ButtonTemplates;
+
+  const uiOptions = getUiOptions(uiSchema);
+  const { RemoveButton } = getTemplate("ButtonTemplates", registry, uiOptions);
```

Comment:
Consider restoring this since we don't need the extra overhead of the call as we can't override them in `uiOptions`. You could add this comment which is in many of the other places (but this one)
```
  // Button templates are not overridden in the uiSchema
```